web/src/stream_events.ts

```diff
@@ -177,6 +181,43 @@ export function update_property<P extends keyof UpdatableStreamProperties>(
             update_stream_setting(sub, value, "is_recently_active");
             stream_list.update_streams_sidebar();
         },
+        is_archived(value) {
+            if (!value) {
+                // We currently do not live-udpate when unarchiving stream.
```

Comment:
`update` has a typo; I'm kinda surprised `tools/run-codespell` didn't catch this.

Reply:
Fixed.